solutions/cluster-api/capi-join.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I think we should rename the script to something else. I put `register-join.sh` above but it can be something else. I don't think the `capi` is necessary.